modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -5080,11 +5080,11 @@ a|<<config_dbms.max_databases,`dbms.max_databases`>>
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|If there is a Database Management System Panic (an irrecoverable error) should the neo4j process shut down or continue running. Following a DbMS panic it is likely that a significant amount of functionality will be lost. Recovering full functionality will require a Neo4j restart. Default is `false` except for Neo4j Enterprise Edition deployments running on Kubernetes where it is `true`.
+a|If there is a server panic (an irrecoverable error) should the neo4j process shut down or continue running. Following a server panic it is likely that a significant amount of functionality will be lost. Recovering full functionality will require the Neo4j process to restart.
```

Comment:
Do we really need the last sentence? I copied it from the code. I know that the PR has been merged already in the monorepo, but could we revisit this description and sync the docs with the code?

Reply:
I'm happy to remove the `This feature is available in Neo4j Enterprise Edition.`, I'll raise a PR for that.

Reply:
I don't like the proposed:
> If there is a server panic (an unrecoverable error), the Neo4j process should shut down or continue running.

Because it's now a definitive statement that says nothing. This setting controls _whether_ the process shuts down or continues running. I would restore the 'should':
> If there is a server panic (an unrecoverable error), should the Neo4j process should shut down or continue running.

Or if it needs restructuring, maybe:
> Controls whether the Neo4j process will shut down if there is a server panic (an unrecoverable error), or continue running.

?

Reply:
https://github.com/neo-technology/neo4j/pull/28708

Reply:
Hi @nick-giles-neo! Thank you for a quick reply. I agree with you. My suggestion is not quite apposite, but I was confused and didn't fully understand how this setting works. I think your second variant is much better. So, I updated my comment.

The original sentence sounds like a question and may confuse user. That's why I started editing it.